docs/cli_commands.md

```diff
@@ -107,6 +107,16 @@ This command lets you configure the behavior of QMK. For the full `qmk config` d
 qmk config [-ro] [config_token1] [config_token2] [...] [config_tokenN]
 ```
 
+## `qmk cd`
+
+This command opens a new shell in your `qmk_firmware` directory.
```

Comment:
```suggestion
This command opens a new shell in your `qmk_firmware` directory. Note that if you are already somewhere within `QMK_HOME` (for example, the `keyboards/` folder), nothing will happen.

To exit out into the parent shell, simply type `exit`.
```

Reply:
May be worth printing a notice to this effect before entering the subshell as well.